web/src/features/charts/bar-breakdown/BarBreakdownChart.tsx

```diff
@@ -121,6 +122,14 @@ function BarBreakdownChart({
     showCapacitySources || showPowerSources || showEmissionSources
   );
 
+  const graphUnit = determineUnit(
+    displayByEmissions,
+    currentZoneDetail,
+    mixMode,
+    timeAverage,
+    t
+  );
```

Comment:
Would it make sense to memo this?

Reply:
I guess it would. I don't know if I did it the best way. If not, then feel free to let me know:)